crates/sui-types/src/sui_system_state/epoch_start_sui_system_state.rs

```diff
@@ -160,11 +160,14 @@ impl EpochStartSystemStateTrait for EpochStartSystemStateV1 {
         self.active_validators
             .iter()
             .filter(|validator| validator.authority_name() != excluding_self)
-            .map(|validator| PeerInfo {
-                peer_id: PeerId(validator.narwhal_network_pubkey.0.to_bytes()),
-                affinity: PeerAffinity::High,
-                address: vec![multiaddr_to_anemo_address(&validator.p2p_address)
```

Comment:
or this?               
                address: multiaddr_to_anemo_address(&validator.p2p_address)
                    .into_iter()
                    .collect(),